cmd/server/cmd/server.go

```diff
@@ -258,10 +260,16 @@ func runServer(sa *serverArgs, printf, fatalf shared.FormatFn) {
 		grpcOptions = append(grpcOptions, grpc.Creds(credentials.NewTLS(tlsConfig)))
 	}
 
+	// setup server grpc interceptors for monitoring
+	grpc_prometheus.EnableHandlingTimeHistogram()
+
 	if sa.enableTracing {
 		tracer := bt.New(tracing.IORecorder(os.Stdout))
 		ot.InitGlobalTracer(tracer)
-		grpcOptions = append(grpcOptions, grpc.UnaryInterceptor(otgrpc.OpenTracingServerInterceptor(tracer)))
+		grpcOptions = append(grpcOptions,
+			grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(otgrpc.OpenTracingServerInterceptor(tracer), grpc_prometheus.UnaryServerInterceptor)))
+	} else {
+		grpcOptions = append(grpcOptions, grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor))
```

Comment:
I think in the CL that added the health/monitoring stuff, there was originally a flag. I believe that I was asked to remove it from that CL.

One worry about adding a flag for this behavior is that we will likely also want to monitor other pieces of the system. I don't want to make that code also be conditional. Another worry is just overloading the cmd with too many flags that default to on.